app/mailers/user_mailer.rb

```diff
@@ -43,22 +39,50 @@ def incoming_interaction_notification_email(client:, user:, received_at:, intera
                elsif interaction_type == "document_upload"
                  "#{@interaction_count} New Document(s) Uploaded by GetYourRefund Client ##{@client_id}"
                end
-    service = MultiTenantService.new(:gyr)
-    attachments.inline['logo.png'] = service.email_logo
+    @unsubscribe_link = Rails.application.routes.url_helpers.url_for(
+      {
+        host: MultiTenantService.new(:gyr).host,
+        controller: "notifications_settings",
+        action: :unsubscribe_from_emails,
+        locale: I18n.locale,
+        _recall: {},
+        email_address: signed_email(@user.email)
+      }
+    )
+    inline_logo
 
-    verifier = ActiveSupport::MessageVerifier.new(Rails.application.secret_key_base)
-    signed_email = verifier.generate(@user.email)
+    mail(to: @user.email, subject: @subject, template_name: "#{interaction_type}_notification_email")
+  end
+
+  def internal_interaction_notification_email(client:, user:, received_at:, interaction_type:)
+    @client_id = client.id
+    @user = user
+    @received_at = received_at.in_time_zone(@user.timezone)
+    @subject = "Tagged in a note for GetYourRefund Client ##{@client_id}"
     @unsubscribe_link = Rails.application.routes.url_helpers.url_for(
       {
         host: MultiTenantService.new(:gyr).host,
         controller: "notifications_settings",
         action: :unsubscribe_from_emails,
         locale: I18n.locale,
         _recall: {},
-        email_address: signed_email
+        email_address: signed_email(@user.email)
       }
     )
+    inline_logo
+
     mail(to: @user.email, subject: @subject, template_name: "#{interaction_type}_notification_email")
   end
 
+  private
+
+  def inline_logo
+    service = MultiTenantService.new(:gyr)
+    attachments.inline['logo.png'] = service.email_logo
+  end
+
+  def signed_email(email)
+    verifier = ActiveSupport::MessageVerifier.new(Rails.application.secret_key_base)
+    verifier.generate(email)
+  end
```

Comment:
i noticed these were duplicated across all the mailer methods so i extracted them here